recipes/configs/lora_finetune_single_device.yaml

```diff
@@ -8,15 +8,16 @@
 # Model Arguments
 model:
   _component_: torchtune.models.llama2.lora_llama2_7b
-  lora_attn_modules: ['q_proj', 'v_proj']
-  apply_lora_to_mlp: False
-  apply_lora_to_output: False
+  lora_attn_modules: ['q_proj', 'v_proj', 'k_proj', 'output_proj']
+  apply_lora_to_mlp: True
+  apply_lora_to_output: True
   lora_rank: 8
   lora_alpha: 16
+  quantize_base: True
```

Comment:
Wonder if this is a case where it makes sense to keep a single recipe and split the configs. E.g. `qlora_finetune_single_device.yaml`. This would also be nice cause QLoRA is one case where we do not (yet) support distributed, so by having only the single-device QLoRA config we make that more explicit. 

Separately, do we want to set `apply_lora_to_mlp` and `apply_lora_to_output` to True by default? No strong opinion on the former, but would consider setting the latter to False, just because it does use quite a bit more mem (even without QLoRA)

Reply:
set `apply_lora_to_output` to False for now. Although one thing to think about is whether our training config should be as close as possible to some sort of reference, the lit-gpt one does have this as true.